tests/testthat/test-maintenanceMetrics.R

```diff
@@ -23,7 +23,7 @@ test_that("Reactivity of maintenanceMetrics", {
   app$wait_for_idle()
   
   # read the current comment -- set to "No comments"
-  out_cmt <- app$get_values()$output$`maintenanceMetrics-view_comments-view_comments`$html
+  out_cmt <- app$get_value(output = "maintenanceMetrics-view_comments-view_comments")$html
```

Comment:
thanks for cleaning these up throughout

Reply:
Yes, they were inconsistent. This should improve upon that.